jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/converter/ssm/SsmTransformer.java

```diff
@@ -0,0 +1,93 @@
+package us.dot.its.jpo.geojsonconverter.converter.ssm;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.kstream.KeyValueMapper;
+import us.dot.its.jpo.asn.j2735.r2024.SignalStatusMessage.SignalStatusMessageMessageFrame;
+import us.dot.its.jpo.geojsonconverter.partitioner.RsuIntersectionKey;
+import us.dot.its.jpo.geojsonconverter.pojos.common.DeserializedRawMessageFrame;
+import us.dot.its.jpo.geojsonconverter.pojos.ssm.ProcessedSsm;
+import us.dot.its.jpo.geojsonconverter.utils.ProcessedSchemaVersions;
+import us.dot.its.jpo.ode.model.OdeMessageFrameData;
+import us.dot.its.jpo.ode.model.OdeMessageFrameMetadata;
+import us.dot.its.jpo.ode.model.OdeMessageFramePayload;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+/**
+ * KeyValueMapper transforms one SSM to one or more ProcessedSsms, for use in flatMap in the Topology.
+ * Implement this as a KeyValueMapper, because the Transformer interface is deprecated.
+ * The transformation is not stateful, so this doesn't need to be a processor.
+ */
+@Slf4j
+public class SsmTransformer
+        implements KeyValueMapper<Void, DeserializedRawMessageFrame, KeyValue<RsuIntersectionKey, ProcessedSsm>> {
+
+    private final SsmConverter converter;
+
+    public SsmTransformer(SsmConverter converter) {
+        this.converter = converter;
+    }
+
+    @Override
+    public KeyValue<RsuIntersectionKey, ProcessedSsm> apply(Void rawKey, DeserializedRawMessageFrame rawSsm) {
+
+        try {
+            if (!rawSsm.isValidationFailure()) {
+                OdeMessageFrameData rawValue = rawSsm.getOdeMessageFrameData();
+                OdeMessageFrameMetadata metadata = rawValue.getMetadata();
+                OdeMessageFramePayload payload = rawValue.getPayload();
+                log.info("payload: {}", payload.getData());
```

Comment:
This log makes this print every message that is processed (that doesn't have validation failures). It is more verbose than the other message types, including SRM.

Reply:
removed this log statement.. It was only for testing during dev